### DISCUSSION TITLE
jpg图片似乎会被重新编码，但部分编码后的图片异常

### DISCUSSION THREAD

**Discussion by westhiker** [messense/aliyundrive-webdav#382] [Q&A] [answered]
### 问题描述

发现webdav访问时，部分图片显示异常，直接去阿里云盘web页面却是正常的。

于是通过浏览器分别从阿里云盘官方网页版及本项目web页面下载图片到本地，然后进行哈希对比，发现并不相同，但二进制程序文件正常。

### 重现步骤

1. 通过阿里云盘官方web上传测试图片 [test-origin](https://user-images.githubusercontent.com/33189692/163100162-503e3807-c45c-42be-a626-8dcc07a03c3c.jpg)
2. 通过**官方**web页面下载这个测试图片，命名为：`test-aliyun`
3. 通过**本项目**web页面下载这个测试图片，命名为：`test-webdav`
4. sha1对比3个文件：
```
>shasum *
5810e9e4b7bf83855b70d9291e849e24287d089b  test-aliyun.jpeg
5810e9e4b7bf83855b70d9291e849e24287d089b  test-origin.jpg
5d0dd97bad4bfe2c0e7dcc545d8280ef5ee22121  test-webdav.jpeg
```
发现并不相同。

附：[test-webdav](https://user-images.githubusercontent.com/33189692/163100680-35c866ce-d52c-40eb-b60e-2b429c4cb48a.jpeg)

### 版本

v1.3.1

### 运行平台

Docker

### 日志

无报错日志

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/discussions/372

**Comment by westhiker** [comment]
之前在issues里面搜索了没有才发的，没想到和讨论里的重复了，sorry。
看那个帖子，貌似这个问题和本项目无关，所以不属于本项目bug。

**Comment by westhiker** [comment]
换回1.3.0版本，此问题消失。

测试了1.3.1，1.3.0，1.2.7，以及另一个Go实现项目(均为docker环境)，只有1.3.1有此问题。 

可以在官方web将此[图片](https://user-images.githubusercontent.com/33189692/163100162-503e3807-c45c-42be-a626-8dcc07a03c3c.jpg)上传，然后在本项目http页面访问测试。 @messense   

另一个兄弟说的都有此问题，也许是浏览器缓存没有刷新的缘故。
